lib/jsdom/browser/Window.js

```diff
@@ -20,6 +20,8 @@ const Navigator = require("../living/generated/Navigator");
 const reportException = require("../living/helpers/runtime-script-errors");
 const { contextifyWindow } = require("./documentfeatures.js");
 
+const GlobalEventHandlersImpl = require("../living/nodes/GlobalEventHandlers-impl").implementation;
```

Comment:
Can't use the interface in non-wrapped classes. I think I may have an idea on how to incrementally move window over, but let's punt on that either to the end of the PR or a new one.